### PR TITLE
Linked List Implementation of Stack

### PR DESCRIPTION
**Pull request by Arun9739**
## Related Issue
- Stack Using Linked List (Issue No. #352 )

## Proposed Changes
- I have uploaded the file which implements stack using linked list

## Additional Info
-  In this program I have used a head as meta data structure for the Stack. 
-  We can see the functions for creating stack , pushing an item into the stack, poping the item from the stack, getting the stack count, getting the stack top, checking the stack is empty etc.
-  The nodes used in the functions have been allocated using the dynamic memory allocation.
-  The program is designed in such a way that we can use it for any data type such as int, float, short, double etc. Because the program has used the void pointer in its all functions.
- Just to demonstrate the concept we have used int data type in main function.

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | 
| ----------- |

| image  | Image goes here     |

### CODE REVIEW

**Comment by miraj0507** [comment]
Hello, @Arun9739 i dont see the readme file with your code, kindly update the readme, and tag me will look at your code after readme.

**Comment by Arun9739** [comment]
@miraj0507 I have updated the Readme accordingly as per your guidelines. I haven't created new Readme.md.  I have just updated the existing stack folder Readme. where my cpp file goes.
Pleasae have a look.

**Comment by Arun9739** [comment]
@siddhi-244 @miraj0507 Please look at my pr

File: S-Stack/README.md [hunk 1]
```diff
@@ -13,4 +13,43 @@ Applications of stack :
...
 3. To reverse a word or a string
 4. _Call stack_ : To store list of functions that are either executing or waiting to be executed during runtime. ( stored in stack in memory)
-5. Practical application : (<-)Back button present in a browser pushes the URLs of the visited sites on a stack.
\ No newline at end of file
+5. Practical application : (<-)Back button present in a browser pushes the URLs of the visited sites on a stack.
+
+
+
+
+# LINKED LIST IMPLEMENTATION OF A STACK
```
Comment (Lakhankumawat) [inline S-Stack/README.md:22]: Change headline to `Linked List Implamentation of Stack`

Comment (Arun9739) [reply S-Stack/README.md]: changed sir

**Comment by Arun9739** [comment]
sir please merge my pr

**Comment by Arun9739** [comment]
@miraj0507 @Lakhankumawat please merge my pr

**Pull request merged**